bin/fetch_ha_env_github.sh

```diff
@@ -9,11 +9,21 @@ ACCESS_TOKEN = ARGV[0]
 
 def fetch_env
```

Comment:
It might an overkill here but have we considered having one script and call it with an argument for release and development? Seems like we will likely be updating both frequently in the future.

Reply:
yeah that would be great, i gave that a pass and struggled a bit to get the github action ci to play nicely with the script args.  They are different enough.

locally we want to pass in the HA_LABEL and fetch the token from the env:
```
bin/fetch_ha_env.sh pr
```

on ci we want to pass in the access token and default to the ha label of github

```
bin/fetch_ha_env.sh $ACCESS_TOKEN
```

i suppose we could pass in both as optional args and find a way to switch, but that struct me as more complicated than simply having two files.  open to any suggestions!

Reply:
also, dotenv is no on the box image by default, so we would also need to standup all of the plumbing to get that working on ci, which is odd as we dont actually need dotenv on ci, just locally.